features.txt

```diff
@@ -51,6 +51,7 @@ regexp-unicode-property-escapes
 
 # Shared Memory and atomics
 # https://github.com/tc39/ecmascript_sharedmem
+Atomics
```

Comment:
Does it really make sense to have feature flags for parts of proposals? Atomics is part of the SharedArrayBuffer proposal. Do we expect implementations to do part of a language feature and separate it out this way?

Reply:
I'm +1 to use a single tag for each proposal, some exceptions can be considered, but in this case I believe it's ok to use the SharedArrayBuffer tag.

Wrt the exceptions, I guess this could possibly apply to private class fields, but we should postpone this discussion to a later time.

Reply:
I added `Atomics` just to be thorough, since it's a new global. Is the consensus here that I should remove?

Reply:
I'm in favor of using a single name for features from the SharedArrayBuffer proposal. Even though, I believe we can change it as a follow up improvement.

If this is good to land, I don't want to block it, we can have a separate discussion to find an agreement for this specific part later.